src/QuickMarcEditor/utils.js

```diff
@@ -285,10 +286,22 @@ export const removeFieldsForDuplicate = (formValues) => {
   };
 };
 
+const checkIsEmptyContent = (field) => {
```

Comment:
Please add tests to verify that fields with no content are removed

Reply:
This case was added to `autopopulateSubfieldSection` test